roles/adr/tasks/main.yml

```diff
@@ -20,7 +20,7 @@
 
 - name: Link adr
   ansible.builtin.file:
-    src: "{{ ansible_env.HOME }}/.local/lib/adr-j-{{ adr_version }}/bin/adr-j"
+    src: "{{ ansible_env.HOME }}/.local/lib/adr-j-{{ adr_adr_version }}/bin/adr-j"
```

Comment:
This is a very weird variable name.

Reply:
Yeah I agree... 🙃 we had a variable adr_version but the convention is to use packagename_variable_name... so if it's the adr_version in the adr package, then it's adr_adr_version ... I think in the moment I was concerned about "version" being ambiguous, but it doesn't look like it in this case so I can probably just change it back to adr_version then